srcpkgs/python3-cairocffi/template

```diff
@@ -1,35 +1,30 @@
 # Template file for 'python3-cairocffi'
 pkgname=python3-cairocffi
-version=1.3.0
-revision=2
+version=1.4.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools python3-cffi python3-wheel $(vopt_if xcb python3-xcffib)"
 depends="python3-cffi cairo"
-checkdepends="python3-pytest python3-numpy gdk-pixbuf $depends"
+checkdepends="python3-pytest python3-numpy gdk-pixbuf python3-pikepdf $depends"
 short_desc="CFFI-based cairo bindings for Python3"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://github.com/Kozea/cairocffi"
 changelog="https://raw.githubusercontent.com/Kozea/cairocffi/master/NEWS.rst"
 distfiles="${PYPI_SITE}/c/cairocffi/cairocffi-${version}.tar.gz"
-checksum=108a3a7cb09e203bdd8501d9baad91d786d204561bd71e9364e8b34897c47b91
+checksum=509339b32ccd8d7b00c2204c32736cde78db53a32e6a162d312478d25626cd9a
 
 build_options=xcb
 case "$XBPS_MACHINE" in
 x86_64*|i686|ppc64le*|ppc64)
 	build_options_default="xcb"
 esac
 
-post_patch() {
-	vsed -e '/pytest-runner/d' -i setup.cfg
-}
-
 do_check() {
-	vsed -e '/addopts/d' -i setup.cfg
-	# Copy in $wrksrc lacks generated module; run against built copy
 	( cd build/lib* && python3 -m pytest )
 }
 
-post_install() {
-	vlicense LICENSE
-}
+# License file was removed for 1.4.0 but will return in next version
+#post_install() {
+#	vlicense LICENSE
+#}
```

Comment:
But the license requires us to distribute the file, doesn't it?

Reply:
I see the manual say `Note: AGPL, MIT, BSD, ISC, X11, and custom licenses require the license file to be supplied with the binary package.` so I will add it.